### ISSUE TITLE
Faceit new client

### ISSUE THREAD

**Issue by Vikingen01** [Jire/Charlatano#481]
Hi! I wonder if the glow still works on the free faceit with the new non-browse client, and if it is undetected.
'

**Comment by blackhat576** [comment]
Yes but the esp is messed up. enemies won't come up as they should but teammates are fine

**Comment by HoopGit** [comment]
Quick question, does that mean that the aimbot will only lock on to visible enemies then? because that would actually be an improvement!

**Comment by attitude12** [comment]
Did you manage to get the cheat working with the anti-cheat enabled?

**Comment by blackhat576** [comment]
Everything works with build 1.0.6. Except for the ESP
Works with the current builds of ESEA CEVO & FACEIT

**Comment by attitude12** [comment]
Nope, it doesn't work if you have the ANTI CHEAT installed and enabled.
